src/main/java/com/purbon/kafka/topology/BuilderCLI.java

```diff
@@ -173,7 +173,7 @@ private void printHelpOrVersion(String[] args) {
       formatter.printHelp(APP_NAME, options);
       exit(0);
     } else if (listOfArgs.contains("--" + VERSION_OPTION)) {
-      System.out.println(KafkaTopologyBuilder.getVersion());
+      System.out.println(JulieOpsBuilder.getVersion());
```

Comment:
should this be called `JulieOps` instead of `JulieOpsBuilder`? 
a benefit of only `JulieOps` is that naming it `*Builder` usually clashes with builder pattern classes.

Reply:
yes, makes a lot of sense! thanks for pointing this our @jeqo !